js/Heap.js

```diff
@@ -23,33 +25,40 @@ const track = (event, payload) => {
   } catch (e) {
     console.log('Error calling Heap.track\n', e);
   }
+});
+
+const triggerCrash = () => {
```

Comment:
Is there a reason we need this in the actual library code vs. the test code?  I'm assuming this is a _test only_ function...

Also, I think it'd probably be funny if I was looking in the source of a new lib and the first thing I saw was a `triggerCrash` function!!!

Reply:
Good point! I wasn't able to find a way to add the `triggerCrash` function to the `Heap` API from the detox level, so I simply moved to unit tests with Jest instead. These probably cover more of the actual code involved anyway, so I'd say this was for the best.